test/e2e/dra/dra.go

```diff
@@ -647,7 +647,7 @@ var _ = ginkgo.Describe("[sig-node] DRA [Feature:DynamicResourceAllocation]", fu
 					framework.Fail("first pod should be running on node, was not scheduled")
 				}
 				framework.ExpectNotEqual(pod1.Spec.NodeName, node, "first pod should run on different node than second one")
```

Comment:
Is it possible to update this line as well ?

```suggestion
gomega.Expect(pod1.Spec.NodeName).ToNot(gomega.Equal(node), "first pod should run on different node than second one")
```